test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Nice one 💯 but as the text in the file states, this file should be ignored by git, hence the mention in the `.gitignore`, so it shouldn't appear here. The reason why it might be appearing here is probably that you pushed it before creating the `.gitignore`. To fix this you should delete the file on GitHub, and pull and recreate it locally, it will be ignored this time. 😉

Reply:
thanks deleted